src/aiida/brokers/__init__.py

```
@@ -6,12 +6,12 @@
 
 from .broker import *
 from .rabbitmq import *
-from .zmq import *
+from .zeromq import *
```

Ruff (0.15.20)

[error] `from .zeromq import *` used; unable to detect undefined names

(F403)